conbench/templates/index.html

```diff
@@ -119,8 +119,12 @@ <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
           "searchPlaceholder": "commit hash works too",
       }
 });
-
 enable_search_query_string('#runs');
 
+/* specify the HTML table id (string) and the indexes of the columns for
+ which search functionality should be enabled in an array format
+*/
+specific_column_search('#runs', [6]);
```

Comment:
forgot to ask - can you add this to the Repository column too? It's theoretically possible to track a few repositories at once in one server.

Reply:
Do we have couple of repos (from where we track), while I searched there was just one. 
Might be tad confusing to have a drop-down with only one option, which does not add any functionality

Reply:
We've already talked about making each column searchable where the search that pops up is relevant to the contents (see below) of the column here and in #613 IMHO we should get that done and shipped in this process (and then we don't have to debate things like this!).

Relevant to the contents:
* < 10 values: a drop down
* < 2 values: nothing? with a small message saying "there's only one value"?
* > 10 values: free text search

That way each column has something to go there, and we can treat each column the same and not need to hand-manage "which column is a free text search? which column is a ...?")